examples/lib/stories/bridge_libraries/flame_jenny/jenny.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:dashbook/dashbook.dart';
+import 'package:examples/stories/bridge_libraries/flame_jenny/commons/commons.dart';
+import 'package:examples/stories/bridge_libraries/flame_jenny/jenny_advanced_example.dart';
+import 'package:examples/stories/bridge_libraries/flame_jenny/jenny_simple_example.dart';
+import 'package:flame/game.dart';
+
+void addJennyExample(Dashbook dashbook) {
+  dashbook.storiesOf('FlameJenny')
```

Comment:
Make sure this follows the same standard as the other bridge libraries

Reply:
I think this is what you mean. [b76db82](https://github.com/flame-engine/flame/pull/3086/commits/b76db828df4e21a13cb988f560e1f338c23091a4)

Reply:
I meant this place, but I see now that this is inline with the others and that it's flame_forge2d that is deviating from the standard